src/NUnitFramework/framework/Constraints/PropertyConstraint.cs

```diff
@@ -23,6 +23,7 @@
 
 using System;
 using System.Reflection;
+
```

Comment:
We don't format our code this way; could you please undo this line?

I think we should consider adding https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/formatting-rules#organize-using-directives to our .editorconfig file because Ctrl+R, Ctrl+G is a useful habit to clean up leftovers, but it behaves differently for different people if our .editorconfig doesn't specify what to do.

Reply:
Same for the two blank lines added in the other file.